release-notes/WIP-cabal-install-3.12.x.0.md

```diff
@@ -0,0 +1,287 @@
+
+    The `cabal path` command prints the file system paths used by Cabal.
+    It is intended for use by tooling that needs to read or modify Cabal
+    data, such that it does not need to replicate the complicated logic
+    for respecting `CABAL_DIR`, `CABAL_CONFIG`, etc.
+
+- `cabal check`: clearly mark Errors [#8908](https://github.com/haskell/cabal/pull/8908)
+
+    `cabal check` will now mark errors (which make the program return 1 and
+    Hackage refuse the package) by prepending them with an "Error: " string in
+    the output.
+
+- Warn when project configuration options are going to be ignored. [#8949](https://github.com/haskell/cabal/pull/8949)
+
+    Some project configuration options can only be specified from the command
+    line.  If the user specified those options from a project file,
+    cabal-install would silently ignore them. Now cabal-install will emit a
+    warning.
+
+- Fix precedence for PATH for build-tools-depends [#8972](https://github.com/haskell/cabal/pull/8972)
+
+    Fixes a bug introduced in #8506 that caused executables in the path to
+    take precedence over those specified in build-tools-depends.
+
+- Remove --cabal-file flags from v2 commands [#6880](https://github.com/haskell/cabal/issues/6880) [#7225](https://github.com/haskell/cabal/issues/7225) [#8395](https://github.com/haskell/cabal/issues/8395) [#9123](https://github.com/haskell/cabal/pull/9123)
+
+    The --cabal-file flag was never meant for public use but only for testing.
+    To avoid confusing the users any further we removed the flag from v2
+    commands.
+
+- Avoid a double space in "Executing install plan ..." [#9376](https://github.com/haskell/cabal/pull/9376)
+
+    The "Executing·install·plan··serially" and other similar "Executing
+    install plan··..." outputs no longer contain double spaces.
+
+- Guard PackageInfo_* modules behind `cabal-version` ≥ 3.12 [#9331](https://github.com/haskell/cabal/issues/9331) [#9481](https://github.com/haskell/cabal/pull/9481)
+
+    `cabal check` now warns whenever PackageInfo_* autogen modules are
+    used with `cabal-version` ≥ 3.12.
+    Additionally, `cabal configure` will fail if you try to use PackageInfo_*
+    with `cabal-version` < 3.12.
+
+- cabal init now generates cabal versions older than 1.12 with the correct >= syntax [#8206](https://github.com/haskell/cabal/issues/8206) [#8860](https://github.com/haskell/cabal/pull/8860)
+- `cabal init`: suggest BSD-3 as default license [#8757](https://github.com/haskell/cabal/issues/8757) [#8764](https://github.com/haskell/cabal/pull/8764)
+- Do not ask overwrite permissions on blank project [#9150](https://github.com/haskell/cabal/issues/9150) [#9155](https://github.com/haskell/cabal/pull/9155)
+- Don't report `index.html` file as created, if not created by Haddock [#5120](https://github.com/haskell/cabal/issues/5120) [#9332](https://github.com/haskell/cabal/pull/9332)
```

Comment:
Another sporadic switch dense and sparse spacing in the Markdown list syntax. Both MD files in this PR  should be checked for that.